apps/x509.c

```diff
@@ -158,6 +160,10 @@ int x509_main(int argc, char **argv)
     BIO *out = NULL;
     CONF *extconf = NULL;
     EVP_PKEY *Upkey = NULL, *CApkey = NULL, *fkey = NULL;
+    int newcert = 0;
+    char *subj = NULL; X509_NAME *fsubj = NULL;
```

Comment:
Please use separate lines for these two declarations.

Reply:
Done